docs/install.md

```diff
@@ -0,0 +1,58 @@
+<!--
+SPDX-FileCopyrightText: 2021 Markus Sommer
+
+SPDX-License-Identifier: GPL-3.0-or-later
+-->
+
+# Installation Instructions
+
+## Manual
+
+- Install the *Go Programming language*, either through your package manager, or from [here](https://golang.org/dl/).
+- Clone the [source repository](https://github.com/dtn7/dtn7-go).
+- Check out the most recent release tag. Or don't and just build the `HEAD`, but we don't promise that it won't be broken.
+- (Optional) Run `GO111MODULE=on go test ./...` in the repository root to make sur we didn't screw up too badly.
+- Run `GO111MODULE=on go build ./...` in the repositry root to build both `dtnd` and `dtn-tool`
```

Comment:
This should only build the source code, but unfortunately does not create the binaries.

```suggestion
- Run `go build ./cmd/dtnd && go build ./cmd/dtn-tool` in the repositry root to build both `dtnd` and `dtn-tool`
```

Reply:
Also `s/repositry/repository/`.